packages/mask/src/extension/popups/pages/Wallet/GasSetting/Prior1559GasSetting.tsx

```diff
@@ -266,8 +275,8 @@ export const Prior1559GasSetting = memo(() => {
                                             formatWeiToEther(gasPrice)
                                                 .times(nativeTokenPrice)
                                                 .times(minGasLimit || 21000),
-                                            'USD',
-                                            { onlyRemainTwoDecimal: true },
+                                            fiatCurrencyType,
+                                            { onlyRemainTwoDecimal: true, fiatCurrencyRate },
```

Comment:
Using a component to replace the pure function, tears the labor of always passing a currency type props.